build-gluster-org/jobs/debian-ubuntu-package.yml

```diff
@@ -0,0 +1,58 @@
+- job:
+    name: debian-package-builder
+    node: master
+    description: glusterfs package script to build both debian and ubuntu
+    project-type: freestyle
+
+    scm:
+    - glusterfs
+
+    parameters:
+    - string:
+        default: v7.0
+        description: Refspec to "git checkout" as the release code snapshot. Typically
+          a git annotated tag like "v6.0". Specifying a branch name will use the
+          current HEAD of that branch.
+        name: GERRIT_REFSPEC
+    - string:
+        default: 7
+        description: Series number for the package to be built against.
+        name: SERIES
+    - string:
+        default: 0
+        description: Version number for the package to be built against.
+        name: VERSION
+    - string:
+        default:
+        description: Release number for the package to be built against.
+          Leave it empty if you are building above series 5.
```

Comment:
Mhh, what is series 5, you mean version 5 branch ? is it still supported, or can we drp it ?

Reply:
We can drop them off.

Reply:
So, the comment is still here, what is series 5 ? Do you mean "release 5" ? (and if we can drop, should it be removed from the description ?)